src/ostorlab/serve_app/agent_groups/sbom.yaml

```diff
@@ -0,0 +1,11 @@
+kind: AgentGroup
+description: SBOM scan
+name: sbom
+agents:
+  - key: agent/ostorlab/osv
+    args:
+      - name: nvd_api_key
+        type: string
+        description: NVD api key.
+        value: ""
```

Comment:
This won't work.
The value should not be set. if it's an empty string, the `apiKey` header will be set to an empty string and the request to the nvd-api call will fail & keep retrying.

Reply:
addressed